packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol

```diff
@@ -153,6 +153,7 @@ contract FullPolicyEscalationManager is BaseEscalationManager, Ownable {
         bool arbitrationResolution
     ) public onlyOwner {
         bytes32 requestId = keccak256(abi.encode(identifier, time, ancillaryData));
+        require(arbitrationResolutions[requestId].valueSet == false, "Arbitration resolution already set");
```

Comment:
Shall we better use shorter revert message to fit in 32 bytes, e.g. `Arbitration already resolved` or `Resolution already set`

Reply:
good call out. have updated accordingly.